packages/session-recorder/src/session-replay/cdn-module.ts

```diff
@@ -15,22 +15,14 @@
  * limitations under the License.
  *
  */
-import { nodeResolve } from '@rollup/plugin-node-resolve'
-import { babel } from '@rollup/plugin-babel'
-import commonjs from '@rollup/plugin-commonjs'
 
-export const babelPlugin = babel({
-	babelHelpers: 'runtime',
-	extensions: ['.js', '.es6', '.es', 'mjs', '.ts'],
-})
+import {
+	SessionReplay as SessionReplayOriginal,
+	SessionReplayPlain as SessionReplayPlainOriginal,
+	// @ts-expect-error Remote module
```

Comment:
This approach (.d.ts) does not work to make types available?

declare module '[session-replay.module.legacy.min.js](https://cdn.signalfx.com/o11y-gdi-rum/session-replay/v1.33.0/session-replay.module.legacy.min.js)' {
....
}

Reply:
It works, thanks for the tip!